datafusion/core/tests/datasource/object_store_access.rs

```diff
@@ -117,8 +117,7 @@ async fn query_multi_csv_file() {
     +---------+-------+-------+
     ------- Object Store Request Summary -------
     RequestCountingObjectStore()
-    Total Requests: 4
-    - LIST prefix=data
+    Total Requests: 3
```

Comment:
it is pretty neat to see this in action

Reply:
These tests have been backed out of the most recent commit because the cache is no longer enabled by default (since it doesn't quite fulfill those requirements yet). Integration level testing changes/enhancements will come once we get the ability to configure the cache from the user level.